### PR TITLE
Nicla Vision Proximity tutorial: Update with clearer installing instructions

### PR DESCRIPTION
**Pull request by rkertesz**
The instructions to install the library were very sparse. I learned much more from the README file in the pololu repo (https://github.com/pololu/vl53l1x-arduino). I pulled similar language into this document to help others who come after me.

## What Needs To Be Reviewed
Some readme language

### CODE REVIEW

File: content/hardware/05.nicla/boards/nicla-vision/tutorials/proximity/content.md [hunk 1]
```diff
@@ -41,7 +41,16 @@ The goals of this project are:
...
-To make sure that the sketch works properly, the latest versions of the **Arduino mbed Core** and the **VL53L1X library** needs to be installed. The **Arduino mbed Core** can be found in the **boards manager...** and the **VL53L1X library** can be found in the **Library manager**, both can be found inside the Arduino IDE.
+To make sure that the sketch works properly, the latest versions of the **Arduino mbed Core** and the **VL53L1X library** needs to be installed. Both can be found inside the Arduino IDE.
+The **Arduino mbed Core** can be found in the **boards manager...**.  
+The **VL53L1X library** can be found in the **Library manager**, 
+
+If you are using version 1.6.2 or later of the Arduino software (IDE), you can use the Library Manager to install the VL53L1X library:
+In the Arduino IDE, open the "Sketch" menu, select "Include Library", then "Manage Libraries...".
+Search for "VL53L1X".
+Click the VL53L1X entry in the list, authored by Pololu.
+Click "Install".
```
Comment (sebromero) [inline content/hardware/05.nicla/boards/nicla-vision/tutorials/proximity/content.md:52]:
```suggestion
Click "Install". More detailed instructions on how to install a library can be found [here](/learn/starting-guide/software-libraries).
```
We have an article that explains how to install a library. I'd take the opportunity and reference it here.

**Review by sebromero — CHANGES_REQUESTED** [review]
Thank you for the contribution. I added a link to an article explaining the installation of libraries.

**Comment by marqdevx** [comment]
> @marqdevx Is this the correct library that is referenced?

Yes, it is also linked inside the meta data of the article.

**Review by jhansson-ard — COMMENTED** [review]
Added some suggestions to the formatting of the text. Let me know if they are acceptable!

**Review by sebromero — APPROVED** [review]
LGTM

**Comment by sebromero** [comment]
@rkertesz Hi Ruben 👋 Thank you for your contribution. If you're happy with the suggestions that we've added we can merge this PR.

**Comment by rkertesz** [comment]
@sebromero changes look good. Thank you

**Review by rkertesz — COMMENTED** [review]
changes improve clarity.

**Pull request merged**